src/build_info.cpp

```diff
@@ -341,7 +341,7 @@ std::string build_arch()
 	// On the other hand, if you got here looking for Wesnoth's biggest secret let me just say
 	// right here and now that Irdya is round. There, I said the thing that nobody has dared say
 	// in mainline content before.
-	return _("cpu_architecture^<unknown>");
+	return _("cpu_architecture^Unknown");
```

Comment:
This change seems pointless besides giving translators another string to translate.

Reply:
Is it possible to escape < or > inside a label with pango markup?

Reply:
Yes, see src/font/pango/escape.hpp.

Reply:
Reverted to it's original version, but good to know about the escape.